features/environment.py

```diff
@@ -7,9 +7,22 @@
 
 
 def before_all(context):
-    os.environ['LD_LIBRARY_PATH'] = (
-        os.getcwd() + '/' + context.config.userdata['shared_libs_path']
-    )
+    # check if we have set relative paths in behave.ini
+    if 'shared_libs_path' in context.config.userdata:
+        shared_libs_path = ''
+        if context.config.userdata['shared_libs_path'][0] == '/':
```

Comment:
Wouldn't `os.path.abspath(context.config.userdata['shared_libs_path'])` achieve the same thing as all this?